astropy/io/ascii/core.py

```diff
@@ -1143,7 +1143,7 @@ def _convert_vals(self, cols):
                             f" column {col.name}, reverting to String.",
                             AstropyWarning,
                         )
-                        col.converters.insert(0, convert_numpy(numpy.str))
+                        col.converters.insert(0, convert_numpy(numpy.str_))
```

Comment:
`io.ascii` maintainers, do you want this or Python built-in `str`?

Reply:
`np.str` was exactly equivalent to the latter. Might be worth investigating at some point if there is a benefit to be get from `np.str_`'s [specific features](https://numpy.org/devdocs/reference/arrays.scalars.html#numpy.str_), but right now it's not clear to me if it will actually work differently from `str` when passed as `dtype`.
Also numpy/numpy#22607 sounds like `np.str_` might be renamed again at some future time.

Reply:
OK, changed. Thanks!